walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,6 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    return "Hello " + name.trim().replaceAll("[^A-Za-z0-9 ]", "");
```

Comment:
nit: can you add a comment of what this does? Only since regexes aren't always the most readable as=is.

Reply:
Done! Thanks for pointing that out